### ISSUE TITLE
Admin allows adding two users with the same username

### ISSUE THREAD

**Issue by mvinent** [sitmun/sitmun-admin-app#222]
**Describe the bug**
The admin allows adding two users with the same username, which should not be permitted and also leads to errors. This means not being able to acces with that username even trying with both passwords created for each instance of the username.

**To Reproduce**
1. Go to admin
2. Go to user screen
3. Create new user with the same name of an existing user.
4. Try to login with that username
5. See error

**Expected behavior**
The admin should check for existing users and not allow repeated usernames. I could show a popup informing the error.

**Comment by fjlopez** [comment]
I am unable to replicate this bug. With an existing user with `username` `admin`, it is not possible to create a new user with the same `username`. In the _Usuari-Nou_ form:

- _Usuari_ (`username`)  is the login name and must be unique.
- _Nom_ (`firstName`) is the first name and can have multiple occurrences

<img width="724" alt="Captura de pantalla 2024-10-15 a las 17 59 08" src="https://github.com/user-attachments/assets/0117a724-60b4-4b8e-958e-326d463b48d5">

@mvinent, could you please provide an example and a screenshot that demonstrates how to reproduce this bug?

**Comment by mvinent** [comment]
You're right in the ['common' deployed app](https://sitmun.github.io/admin-app)  it's working fine, as you say. We've detected this bug in our [own deployment of the app](https://idetest.cime.es/stm3/admin), it's possible that we have deployed an older version.

![image](https://github.com/user-attachments/assets/17f2983d-0210-41a9-b449-2a88a502ccfb)


We'll donwload the latest version and we'll test it again.
